src/Lecture1.hs

```diff
@@ -53,8 +54,9 @@ their squares.
 Explanation: @sumOfSquares 3 4@ should be equal to @9 + 16@ and this
 is 25.
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-sumOfSquares x y = error "TODO!"
+
+sumOfSquares :: Num a => a -> a -> a
+sumOfSquares x y = x ^ (2 :: Int) + y ^ (2 :: Int)
```

Comment:
This is the correct solution 👍🏻

Another approach is to use `*` in this particular case to avoid ambiguity imposed by the `^ 2` operator application:

```suggestion
sumOfSquares x y = x * x + y * y
```